ccmbioinfo/lfs
# CARDS-2406: Add Survey Events questions for tracking IC notification emails

- Add 3 new questions to the Survey Events form for initial, reminder 1 and reminder 2 emails
- Update the Email Notification task to point to these new questions rather than the previous non-IC questions

To test:
- `mvn clean install`
- Launch cards using `NIGHTLY_NOTIFICATIONS_SCHEDULE="0 * * ? * *" ./start_cards.sh --project cards4prems --dev`
- Create a new patient who can receive emails
- Create a new visit information form for that patient that occurred 30 days ago
- Save and return to dashboard
- Wait until a new minute starts and then open the Survey Events form for that patient. Observe that the Integrated Care Invitation question has been filled out.
- Edit the visit information for to be 37 days ago. Save + close and wait a minute
- Observe that the Survey Events Integration Care first reminder question has been filled out
- Edit the visit information for to be 44 days ago. Save + close and wait a minute
- Observe that the Survey Events Integration Care second reminder question has been filled out

![image](https://github.com/data-team-uhn/cards/assets/20056751/ddc13456-bea4-4794-bb89-140c2abfd57b)
